nucypher/network/nodes.py

```diff
@@ -558,21 +556,14 @@ def keep_learning_about_nodes(self, learner=None, frames=None):
         self._learning_deferred = Deferred(canceller=self._discovery_canceller)  # TODO: No longer relevant.
 
         def _discover_or_abort(_first_result):
-            self.log.debug(f"========={self} learning at {datetime.datetime.now()}")
+            self.log.debug(f"{self} learning at {datetime.datetime.now()}")
             result = self.learn_from_teacher_node(eager=False, canceller=self._discovery_canceller)
-            self.log.debug(f"///////////{self} finished learning at {datetime.datetime.now()}")
+            self.log.debug(f"{self} finished learning at {datetime.datetime.now()}")
             return result
 
         self._learning_deferred.addCallback(_discover_or_abort)
         self._learning_deferred.addErrback(self.handle_learning_errors)
 
-        # def clear_learning_deferred(result_of_last_learning_cycle):
```

Comment:
What's the basic motive for removing these two blocks?

Reply:
It just never really worked.  The `RELAX` pattern (which in compsci terms is, I believe, a "callback cancellation" pattern) more-or-less renders this obsolete.  Though I may take a swing at it when we get around to optimizing high-throughput retrieve.